From dd0fea71b12bd016f5c3fb05fd2d9f999cc281b5 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 2 Sep 2015 15:47:32 +0900
Subject: [PATCH 1467/2066] arm64: dts: r8a7795-es1: Add GSX device node

This patch adds GSX device node for R8A7795 SoC.

[phat.le.wh: change domain name of GSX from 3DG to 3DGE]
Signed-off-by: Phat Le <phat.le.wh@rvc.renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index a4e4356..aeb39f9 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -1757,6 +1757,14 @@
 			};
 		};
 
+		gsx: gsx@fd000000 {
+			compatible = "renesas,gsx";
+			reg = <0 0xfd000000 0 0x3ffff>;
+			interrupts = <GIC_SPI 119 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&cpg CPG_MOD 112>;
+			power-domains = <&sysc R8A7795_PD_3DG_E>;
+		};
+
 		vin0: video@e6ef0000 {
 			compatible = "renesas,vin-r8a7795";
 			reg = <0 0xe6ef0000 0 0x1000>;
-- 
1.7.5.4

